### PR TITLE
New package at r-spades at 2.0.6

### PR DESCRIPTION
**Pull request by dorton21**

### CODE REVIEW

**Comment by dorton21** [comment]
depends on #21783

**Comment by dorton21** [comment]
depends on #21782

**Comment by dorton21** [comment]
depends on #21781

File: var/spack/repos/builtin/packages/r-spades/package.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class RSpades(RPackage):
+    """Develop and Run Spatially Explicit Discrete Event Simulation Models"""
+
```
Comment (glennpj) [inline var/spack/repos/builtin/packages/r-spades/package.py:15]: Include the description.

File: var/spack/repos/builtin/packages/r-spades/package.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    version('2.0.6', sha256='0fa59d1737c67abeb04eae894939bc4700f92d6c2cc2ec3489b4650720ede5a3')
+
+    depends_on('r@3.6:', type=('build', 'run'))
+    depends_on('r-quickplot', type=('build', 'run'))
+    depends_on('r-reproducible@1.2.1.9007:', type=('build', 'run'))
+    depends_on('r-spades-addins', type=('build', 'run'))
+    depends_on('r-spades-core@1.0.4:', type=('build', 'run'))
+    depends_on('r-spades-tools', type=('build', 'run'))
+    depends_on('r-codetools', type=('build', 'run'))
```
Comment (glennpj) [inline var/spack/repos/builtin/packages/r-spades/package.py]: This dependency is not listed and should be removed.

**Review by glennpj — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/r-spades/package.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class RSpades(RPackage):
+    """Develop and Run Spatially Explicit Discrete Event Simulation Modelsi
```
Comment (glennpj) [inline var/spack/repos/builtin/packages/r-spades/package.py]:
```suggestion
    """Develop and Run Spatially Explicit Discrete Event Simulation Models
```

**Review by glennpj — CHANGES_REQUESTED** [review]

**Review by glennpj — APPROVED** [review]

**Pull request merged**